src/exm-rating.c

```diff
@@ -143,4 +143,4 @@ exm_rating_init (ExmRating *self)
     gtk_widget_init_template (GTK_WIDGET (self));
 
     update_rating (self);
-}
+}
```

Comment:
Please don't remove the newline here as it pollutes git history. You can remove this file altogether using:

```
# Undo last commit(s) where 1 is the number of commits to undo
git reset --soft HEAD~1

# Add only the relevant files
git add src/exm.gresource.xml
git add data

# Recommit
git commit -m "..."
```